test/isArgument_test.js

```diff
@@ -0,0 +1,34 @@
+goog.require('helper');
+
+describe('The helper.isArguments_ function', function() {
+
+  it('Identifies an arguments object', function() {
+      const argumentsTest = () => {
+        expect(helper.isArguments_(arguments)).toBe(true);
+      };
+      argumentsTest(1, 2, 3, 4);
```

Comment:
You should test a case other than just 1,2,3,4, like if you pass in no arguments or use rest parameters or destructuring assignment.